src/lib.rs

```diff
@@ -1,14 +1,363 @@
-pub fn add(left: u64, right: u64) -> u64 {
-    left + right
+use std::hash::{DefaultHasher, Hash, Hasher};
+
+/// Returns the hash of a single `Hash` value.
```

Comment:
It seems that the documentation says that the function returns the hash of a hash.

```suggestion
/// Returns the hash of a single value.
```

or

```suggestion
/// Returns the hash of a single value. The value type must implement the `Hash` trait.
```

Reply:
Updated.